tests/suites/test_suite_mpi.function

```diff
@@ -2140,11 +2140,22 @@ void mpi_core_montmul( int limbs_AN4, int limbs_B4,
 
     memcpy( N.p, R.p, bytes );
 
-    /* The output may even be aliased to B, if AN_limbs == B_limbs */
-
     if (limbs_AN == limbs_B)
     {
-        /* Note: last test, so we don't save B */
+        /* Test when A aliased to B (requires A == B on input values) */
+        if ( memcmp( A.p, B.p, bytes ) == 0 )
+        {
+            /* Test with A aliased to B and output, since this is permitted -
+             * don't bother with yet another test with only A and B aliased */
```

Comment:
Please do bother with a test with X==A and B not aliased and not necessarily the same size, and with X==B!=A. Aliasing-related bugs are often hard to track down, it's worth writing a few more lines of test code.

Reply:
Tested aliasing: X == A != B, X != A == B, X == A == B and X == N. What is not tested is X != A == B, because it seems very unlikely that if X == A == B works, that disjoint X would not work with A == B.

Reply:
Ah, my bad, I was too hasty in my differential review and didn't check that X==A!=B and X==B!=A were already covered by another commit. I don't mind not testing aliasing of inputs that aren't aliased to any ouptut: we don't do anything that could make it problematic (such as shared memory or value merging — not the kind of thing you'd ever do in a crypto library).